modules/installation-creating-aws-vpc-subnets-edge.adoc

```diff
@@ -0,0 +1,86 @@
+* You added your AWS keys and region to your local AWS profile by running `aws configure`.
+* You opted in to the {zone-type} group.
+
+.Procedure
+
+. Go to the section of the documentation named "CloudFormation template for the VPC subnet", and then copy the syntax from the provided template. Save the copied template syntax as a YAML file on your local system. This template describes the VPC that your cluster requires.
+
+. Run the following command to deploy the CloudFormation template that in turn creates a stack of AWS resources that represent the VPC. You must enter the command on a single line. For a Local Zones environment, do not specify the public route table parameter in the command.
++
+[TIP]
+====
+You must enter the VPC ID in the command for creating a stack of AWS resources. The following example command demonstrates outputting the VPC ID value:
+
+[source,terminal]
+----
+VPC_ID=$(aws cloudformation describe-stacks --stack-name "<vpc_stack_name>" \
+  --query 'Stacks[].Outputs[?OutputKey==`VpcId`].OutputValue' --output text)")
+----
+====
++
+[source,terminal]
+----
+aws cloudformation create-stack --stack-name <stack_name> \ <1>
+  --region ${CLUSTER_REGION} \
+  --template-body file://<template>.yaml \ <2>
+  --parameters \
+    ParameterKey=VpcId,ParameterValue="${VPC_ID}" \ <3>
+    ParameterKey=ClusterName,ParameterValue="${CLUSTER_NAME}" \ <4>
+    ParameterKey=ZoneName,ParameterValue="${ZONE_NAME}" \ <5>
+    ParameterKey=PublicRouteTableId,ParameterValue="${ROUTE_TABLE_PUB}" \ <6>
+    ParameterKey=PublicSubnetCidr,ParameterValue="${SUBNET_CIDR_PUB}" \ <7>
+    ParameterKey=PrivateRouteTableId,ParameterValue="${ROUTE_TABLE_PVT}" \ <8>
+    ParameterKey=PrivateSubnetCidr,ParameterValue="${SUBNET_CIDR_PVT}" <9>
+----
+<1> `<stack_name>` is the name for the CloudFormation stack, such as `cluster-wl-<local_zone_shortname>` for Local Zones and `cluster-wl-<wavelength_zone_shortname>` for Wavelength Zones. You need the name of this stack if you remove the cluster.
+<2> `<template>` is the relative path and the name of the CloudFormation template
+YAML file that you saved.
+<3> `${VPC_ID}` is the VPC ID, which is the value `VpcID` in the output of the CloudFormation template for the VPC.
+<4> `${ZONE_NAME}` is the value of {zone-type} name to create the subnets.
+<5> `${CLUSTER_NAME}` is the value of *ClusterName* to be used as a prefix of the new AWS resource names.
+<6> Wavelength only: `${ROUTE_TABLE_PUB}` is the *PublicRouteTableId* extracted from the output of the VPC's carrier gateway CloudFormation stack.
+<7> `${SUBNET_CIDR_PUB}` is a valid CIDR block that is used to create the public subnet. This block must be part of the VPC CIDR block `VpcCidr`.
+<8> `${ROUTE_TABLE_PVT}` is the *PrivateRouteTableId* extracted from the output of the VPC's CloudFormation stack.
+<9> `${SUBNET_CIDR_PVT}` is a valid CIDR block that is used to create the private subnet. This block must be part of the VPC CIDR block `VpcCidr`.
```

Comment:
- invert the order of 4 and 5
- public rtb is not exclusively for WL, it must be used for both, but extracted from different sources (stacks)
```suggestion
<4> `${CLUSTER_NAME}` is the value of *ClusterName* to be used as a prefix of the new AWS resource names.
<5> `${ZONE_NAME}` is the value of {zone-type} name to create the subnets.
<6> `${ROUTE_TABLE_PUB}` is the *PublicRouteTableId* extracted from the CloudFormation template. For Local Zones, the public Route Table should be extracted from the VPC CloudFormation Stack. For Wavelength: the value must be extracted from the output of the VPC's carrier gateway CloudFormation stack.
<7> `${SUBNET_CIDR_PUB}` is a valid CIDR block that is used to create the public subnet. This block must be part of the VPC CIDR block `VpcCidr`.
<8> `${ROUTE_TABLE_PVT}` is the *PrivateRouteTableId* extracted from the output of the VPC's CloudFormation stack.
<9> `${SUBNET_CIDR_PVT}` is a valid CIDR block that is used to create the private subnet. This block must be part of the VPC CIDR block `VpcCidr`.
```

Reply:
Thanks, updated.